crud/common/utils.lua

```diff
@@ -97,6 +97,20 @@ end
 
 function utils.get_space(space_name, replicasets)
     local replicaset = select(2, next(replicasets))
+
+    if replicaset.master == nil then
+        local error_msg = string.format(
+            'The master is not configured for replicaset %s, ' ..
+            'check status of the master and repeat the operation later',
+             replicaset.uuid)
+        error(error_msg)
+    end
+    if replicaset.master.conn.error ~= nil then
+        local error_msg = string.format(
+            'The connection to the master of replicaset %s is not valid: %s',
+             replicaset.uuid, replicaset.master.conn.error)
+        error(error_msg)
+    end
     local space = replicaset.master.conn.space[space_name]
```

Comment:
Have you considered a possibility to test it? (I don't insist, but maybe there are some good recipes in vshard tests already).